ApiStubGen/Scripts/README.MD

```diff
@@ -0,0 +1,14 @@
+# Create API for Package Indexing
+
+This script can be used to upload a package to APIView to generate tokens. APIView is a web app running on Azure and purpose of this web app to is generate information about public APIs in a package. We are utilizing this tool for package indexing since it already has a builtin mechanism to generate tokens for all publicly accessible APIs.
+
+# How does it work
+
+Script uploads given package to APIView instance which internally stores token file in azure storage account. APIView returns URL to created review if a package is processed successfully. This URL has review ID which is used to find review and blob details from COSMOS DB where review details are stored. Cosmos DB item has details like revision ID and file ID which points to blob that stores parsed tokens.
```

Comment:
> This URL has review ID which is used to find review and blob details from COSMOS DB where review details are stored.

The one open question I had was re: this level of indirection and if the initial Post could point us at storage?  if this isn't feasible I totally can follow, it's just the one question that comes to mind for why it's a two-hop. (and for why it requires an additional set of creds)